docs/install-olm.md

```diff
@@ -0,0 +1,64 @@
+# How do I install OLM?
+
+The OLM is installable on Kubernetes clusters. It is already installed and available by default on 4.x OpenShift 
+clusters. For the following instructions to work, you must have a Kubernetes cluster running and the `kubectl` 
+command-line tool is able to find the information it needs to communicate with the API server of that cluster. For more 
+information about configuring `kubectl`, please visit 
+[here](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+
+### Install locally with minikube
+
+To install OLM locally on a minikube cluster, you need to spawn a 1.16.0 minikube cluster and then follow the 
+instructions installing OLM from releases or GitHub repositories:
+
+```bash
+minikube start  --kubernetes-version=1.16.0 
+```
+
+## Install Released OLM
+For installing release versions of OLM, for example version 0.12.0, you can use the following command:
+
+```bash
+export olm_release=0.12.0
+kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/olm.yaml
+```
+
+For detailed install instructions of individual releases, please visit the 
+[latest release on GitHub](https://github.com/operator-framework/operator-lifecycle-manager/releases).
+
+## Install From Git Repository
+
+You can install OLM from the master branch of the 
+[operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager/) 
+repository with the following: 
+
+```bash
```

Comment:
Should we be telling admins to download the git repo? Instead, we could provide the following commands to deploy master:
```bash
kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
```